adapter/src/main/java/com/twilio/kudu/sql/KuduPrepareImpl.java

```diff
@@ -159,15 +159,17 @@ public void executeDdl(Context context, SqlNode node) {
         PartialRow lowerBound = tableSchema.newPartialRow();
         PartialRow upperBound = tableSchema.newPartialRow();
 
-        // Set range partition to EPOCH.MAX - minvalue for DESC case.
-        if (isRowTimeStampColDesc.get()) {
-          lowerBound.addTimestamp(rowTimestampColumn,
-              new Timestamp(CalciteKuduTable.EPOCH_FOR_REVERSE_SORT_IN_MILLISECONDS - Long.MIN_VALUE));
-          upperBound.addTimestamp(rowTimestampColumn,
-              new Timestamp(CalciteKuduTable.EPOCH_FOR_REVERSE_SORT_IN_MILLISECONDS - Long.MIN_VALUE + 1));
-        } else {
-          lowerBound.addTimestamp(rowTimestampColumn, new Timestamp(Long.MIN_VALUE));
-          upperBound.addTimestamp(rowTimestampColumn, new Timestamp(Long.MIN_VALUE + 1));
+        if (kuduSchema.createDummyPartition) {
```

Comment:
Adds a flag that determines whether we create a single dummy partition for the timestamp column or if we have an unbounded range partition.